### ISSUE TITLE
All `export * as Foo from "./file"` declarations are ignored

### ISSUE THREAD

**Issue by RunDevelopment** [timocov/dts-bundle-generator#139]
**Bug report**
As the title says, all `export * as Foo from "./file"` declarations are ignored, kinda. All that gets generated is an empty `export {};`. `export * from "./file"` works fine.

**Input code**

```ts
// index.ts
export interface Foo {
	foo: number;
}

export * as Bar from "./bar";

// ./bar.ts
export interface Bar {
	bar: number;
}
```

**Expected output**

```ts
// Generated by dts-bundle-generator v5.4.0

export interface Foo {
	foo: number;
}

export namespace Bar {
	interface Bar {
		bar: number;
	}
};
```

**Actual output**

```ts
// Generated by dts-bundle-generator v5.4.0

export interface Foo {
	foo: number;
}

export {};
```

**Additional context**
[Here is a minimal project](https://github.com/timocov/dts-bundle-generator/files/5391448/foo.zip) reproducing the issue. Run `npm ci && npm run dts-bundle` to generate the `index.d.ts` bundle.

**Comment by timocov** [comment]
See https://github.com/timocov/dts-bundle-generator/issues/134#issuecomment-699616069, maybe you have answers for questions from that comment :)

**Comment by timocov** [comment]
Closed in favor of #134
